readthedocs/rtd_tests/tests/test_integrations.py

```diff
@@ -62,15 +71,17 @@ def test_exchange_form_request_body(self):
         integration = fixture.get(Integration, project=project,
                                   integration_type=Integration.GITHUB_WEBHOOK,
                                   provider_data='')
-        resp = client.post(
+        payload_decode = '{"head_commit": {"id": "3eea78b2", "message": "Update README.md"}, "ref": "exchange_form"}'
+        payload_encode = '%7B%22head_commit%22%3A%20%7B%22id%22%3A%20%223eea78b2%22%2C%20%22message%22%3A%20%22Update%20README.md%22%7D%2C%20%22ref%22%3A%20%22exchange_form%22%7D'
```

Comment:
I use this tool for encode/decode https://meyerweb.com/eric/tools/dencoder/

Note: the order matters!

Reply:
Can you do this with a Python function to encode a JSON into an URL query string? (it should be there one in the standard library)

Reply:
Sure, I think that's cleaner, just wanted to keep to the current tests style.